modules/ROOT/pages/getting-started.adoc

```diff
@@ -0,0 +1,60 @@
+= Getting Started with Fedora/CentOS bootc
+
+== Introduction
+
+The Fedora/CentOS bootc project generates reference "base images" that
+are designed for use with the https://github.com/containers/bootc[bootc project].
+
+=== Container Images
+
+These are the two "full" container images:
+
+- {container-c9s} (CentOS Stream 9)
+- {container-fedora-full} (Fedora 40)
+
+=== Provisioning Philosophy
+
+It is generally expected that you create a *custom* container image, derived from one of our base images for deployment.
+
+For more information on configuration, refer to the documentation for xref:building-containers.adoc[Building containers].
+
+== Quickstart
+
+=== Running as a container
+
+include::getting-started-container.adoc[]
+
+=== Booting on a local hypervisor
+
+include::getting-started-local-virt.adoc[]
+
+=== Exploring the OS for the first time
+
+By design, there are no hard-coded default credentials or non-root interactive users.
+
+If you set up an xref:authentication.adoc[SSH key], you can SSH into the VM and explore the OS:
```

Comment:
this link takes me to a page that doesn't mention SSH keys at all. I know when you configure users you can set an SSH key, but the connection isn't direct which makes the flow not great.

examples on that page might help.

Reply:
This seems to have been addressed in a new upload.